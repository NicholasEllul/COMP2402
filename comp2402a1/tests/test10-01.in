 4
 2
 7
 9
 0
 8
 1
 8
 4
 3
 6
 3 
