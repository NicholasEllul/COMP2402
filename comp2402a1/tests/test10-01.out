 9
 8
 8
 6
 3 
